### DISCUSSION TITLE
ffmpeg:   hwaccel_args: preset-nvidia

### DISCUSSION THREAD

**Discussion by MrCaspan** [blakeblackshear/frigate#15030] [Ask A Question] [answered]
What is the difference between the hardware acceleration option `preset-nvidia` and `preset-nvidia-h264` is there a reason why I would chose one over the other? or are they jsut aliases to the same preset?  I have a Nvidia GTX 1080 just FYI

**Comment by NickM-27** [comment, marked as answer]
They are just aliases, they are identical
